### PR TITLE
Enhancement : Remove Kitkat API Check & Fix variables

### PR DESCRIPTION
**Pull request by salRoid**

### CODE REVIEW

File: app/src/main/java/tech/salroid/filmy/activities/FullCrewActivity.java [hunk 1]
```diff
@@ -92,28 +88,14 @@ protected void onCreate(Bundle savedInstanceState) {
 
     @Override
     public void itemClicked(CrewDetailsData setterGetter, int position, View view) {
-
         Intent intent = new Intent(this, CharacterDetailsActivity.class);
         intent.putExtra("id", setterGetter.getCrewId());
-
-        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.KITKAT) {
```
Comment (webianks) [inline app/src/main/java/tech/salroid/filmy/activities/FullCrewActivity.java:99]: Just remove the check, but keep the block

File: app/src/main/java/tech/salroid/filmy/activities/FullCastActivity.java [hunk 2]
```diff
@@ -94,19 +94,7 @@ protected void onCreate(Bundle savedInstanceState) {
     public void itemClicked(CastDetailsData setterGetter, int position, View view) {
         Intent intent = new Intent(this, CharacterDetailsActivity.class);
         intent.putExtra("id", setterGetter.getCastId());
-
-        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.KITKAT) {
```
Comment (webianks) [inline app/src/main/java/tech/salroid/filmy/activities/FullCastActivity.java:98]: Please keep the if block content just remove the if check and else (along with block)

**Review by webianks — CHANGES_REQUESTED** [review]

File: app/src/main/java/tech/salroid/filmy/activities/FullCastActivity.java (hunk 2 above)
Comment (salRoid) [reply app/src/main/java/tech/salroid/filmy/activities/FullCastActivity.java:98]: Updated

File: app/src/main/java/tech/salroid/filmy/activities/FullCrewActivity.java (hunk 1 above)
Comment (salRoid) [reply app/src/main/java/tech/salroid/filmy/activities/FullCrewActivity.java:99]: Updated

**Review by webianks — APPROVED** [review]

**Pull request merged**